### PR TITLE
Pro 6439 document reprint

### PR DESCRIPTION
**Pull request by sanjayparekh-hmcts**
add doc reprint functionality

### CODE REVIEW

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -126,6 +126,7 @@
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boRedeclarationComplete", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "assembleLetter", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "assembleLetterForReissue", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "reprintDocument", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
```
Comment (jackmunkonge) [inline ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json:129]: Why is only issuer given permission?

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -410,7 +410,7 @@ idam:
     microservice: probate_backend
 
 send-letter:
-  url: ${SEND_LETTER_SERIVCE_BASEURL:http://localhost:8485}
+  url: ${SEND_LETTER_SERIVCE_BASEURL:http://rpe-send-letter-service-aat.service.core-compute-aat.internal/}
```
Comment (jackmunkonge) [inline src/main/resources/application.yml]: Are you sure this is okay? I thought we weren't using internal URLs anymore

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json (hunk 1 above)
Comment (sanjayparekh-hmcts) [reply ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json:129]: Good spot - ill add that

File: src/main/resources/application.yml (hunk 2 above)
Comment (sanjayparekh-hmcts) [reply src/main/resources/application.yml]: sorry - needs to be reverted anyway

**Review by jackmunkonge — APPROVED** [review]

**Review by ruban72 — APPROVED** [review]

**Pull request merged**